### PR TITLE
Feature/reviews requests page

### PR DESCRIPTION
**Pull request by LiveBacteria**
Functionality and error catching successful.

### CODE REVIEW

File: client/src/services/reviews.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  //Make get call to api for retrieving reviews on current user
+  const config = {
+    method: "post",
+    url: "/api/user/reviews",
+    data: {
+      user: "5f20c02f42b72242883a7674",
+      singleTarget: true,
+      reviewId: reviewId,
+    },
+  };
```
Comment (skassam21) [inline client/src/services/reviews.js:38]: Are you hardcoding the fetching of reviews right now? This should probably be something that we definitely not aim to hardcode

File: client/src/pages/Reviews.js [hunk 2]
```diff
@@ -1,25 +1,235 @@
...
+      "Sep",
+      "Oct",
+      "Nov",
+      "Dec",
+    ];
+    var d = date.getDate();
+    var m = strArray[date.getMonth()];
+    var y = date.getFullYear();
+    return "" + (d <= 9 ? "" + d : d) + " " + m + " " + y;
+  }
```
Comment (skassam21) [inline client/src/pages/Reviews.js:142]: This should be written in a utils file and not in the component itself

File: client/src/elements/Background.js [hunk 3]
```diff
@@ -1,21 +1,43 @@
...
 }));
 
 export default function Background(props) {
   const classes = useStyles();
-  return <div className={props.gradient ? classes.gradient : classes.solid}>{props.children}</div>;
+
+  // Need to implement switch for in app themes
+  const getBackroundType = (type) => {
+    const backgrounds = {};
+  };
```
Comment (skassam21) [inline client/src/elements/Background.js:36]: I don't think you are using this at the moment right? Don't know exactly it's purpose at the moment

File: client/src/pages/Reviews.js (hunk 2 above)
Comment (LiveBacteria) [reply client/src/pages/Reviews.js:142]: I will move this code later tonight.

File: client/src/elements/Background.js (hunk 3 above)
Comment (LiveBacteria) [reply client/src/elements/Background.js:36]: Will remove from code, as yes this is not used.

File: client/src/services/reviews.js (hunk 1 above)
Comment (LiveBacteria) [reply client/src/services/reviews.js:38]: Can you elaborate on how this is hard-coded?

From the way it looks, its being passed variables reviewId as a string and singleTarget as a boolean.

Comment (LiveBacteria) [reply client/src/services/reviews.js:38]: Resolved

File: client/src/elements/Background.js (hunk 3 above)
Comment (LiveBacteria) [reply client/src/elements/Background.js:36]: Removed

File: client/src/pages/Reviews.js (hunk 2 above)
Comment (LiveBacteria) [reply client/src/pages/Reviews.js:142]: Moved

**Pull request merged**